### PR TITLE
Refactors face rectangle to face center point

### PR DESCRIPTION
**Pull request by zcstarr**
Moves faces to faces_centered and returns the center point of the face

### CODE REVIEW

**Comment by zcstarr** [comment]
@aboodman  PTAL

File: samples/js/fb/find-photos/src/main.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ async function main(): Promise<void> {
         photo.geoposition = getGeo(v);
       }
       if (isSubtype(tagsType, v.type)) {
-        photo.faces = await getFaces(v);
+        photo.faces_centered = await getFaces(v);
```
Comment (aboodman) [inline samples/js/fb/find-photos/src/main.js]: We use `camelCase` for noms struct fields.

File: samples/js/fb/find-photos/src/main.js [hunk 2]
```diff
@@ -142,11 +142,9 @@ async function getFaces(photo): Promise<Set<Struct>> {
   const mh = 0.33;
```
Comment (aboodman) [inline samples/js/fb/find-photos/src/main.js]: This constant unused now. Run `npm test` to find this stuff.

**Comment by aboodman** [comment]
lgtm

**Pull request merged**